File: Examples/Example-ObjC/Example-ObjC/ViewController.m
//
//  ViewController.m
//  Example-ObjC
//
//  Created by Samed Düzçay on 22/09/2020.
//  Copyright © 2020 Infoset Teknoloji A.S. All rights reserved.
//

#import "ViewController.h"
@import Infoset;
@import MapKit;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.title = @"Infoset";
}

- (IBAction)openChat:(id)sender {
    //Presenting chat:
    [InfosetChat presentChatWithAnimated:YES completion:nil];
}

- (IBAction)clearSession:(id)sender {
    //Clearing session:
    [InfosetChat clearSession];
}

@end

File: Examples/Example-Swift/Example-Swift/ViewController.swift
//
//  ViewController.swift
//  Example-Swift
//
//  Created by Samed Düzçay on 22/09/2020.
//  Copyright © 2020 Infoset Teknoloji A.S. All rights reserved.
//

import UIKit
import Infoset
import MapKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        title = "Infoset"
    }
    
    @IBAction func openChat(_ sender: Any) {
        //Presenting chat:
        InfosetChat.presentChat()
    }
    
    @IBAction func clearSession(_ sender: Any) {
        //Clearing session:
        InfosetChat.clearSession()
    }
}

